src/Symfony/Component/Workflow/EventListener/GuardListener.php

```diff
@@ -77,7 +80,13 @@ private function getVariables(GuardEvent $event): array
         $token = $this->tokenStorage->getToken();
 
         if (null === $token) {
-            throw new InvalidTokenConfigurationException(sprintf('There are no tokens available for workflow "%s".', $event->getWorkflowName()));
+            if (!$this->useAuthenticatorManager) {
+                throw new InvalidTokenConfigurationException(sprintf('There are no tokens available for workflow "%s".', $event->getWorkflowName()));
+            }
+            if (!class_exists(NullToken::class)) {
```

Comment:
Instead of the check, can't we rely on the fact that useAuthenticatorManager is set to true only when the NullToken class exists? WHat about doing this check in the DI extension? I'm not comfortable with telling about a DI parameter in a standalone component...